### ISSUE TITLE
Attempting to run most recent update and failing with Yarn error

### ISSUE THREAD

**Issue by astrugatch** [moonmeister/xo-install-script#5]
Running the update script to get the latest Xen orchestra (has worked fine in the past) and I am getting the following error.

```
Updating f5febef..f20604d
error: Your local changes to the following files would be overwritten by merge:
	yarn.lock
Please, commit your changes or stash them before you can merge.
```

**Comment by moonmeister** [comment]
the `yarn.lock` probably needs to be added to the .gitignore. I can do it on master here quickly and you can pull or just do it manually.

**Comment by moonmeister** [comment]
should be fixed with 371a83e let me know if you have any more troubles.

**Comment by moonmeister** [comment]
I just got this issue on an update. looks like xo needs to include the yarn.lock in their .gitignore. @olivierlambert can you shed light on this? Any reason this hasn't been done already? 

In the meantime you can do a `git checkout yarn.lock` to reset it to the default version then re-run the update.
